api/v1/misc.js

```diff
@@ -7,17 +7,23 @@ const scanner = require('portscanner');
 
 const { getYells } = require('./utils/yells');
 
-router.get('/status', (req, res) => {
+router.get('/status', async (req, res) => {
     try {
-        scanner.checkPortStatus(54231, 'localhost', async (error, status) => {
-            if (!error && status === 'open') {
-                const query = 'SELECT COUNT(*) AS ct FROM accounts_sessions JOIN chars ON accounts_sessions.charid = chars.charid WHERE gmlevel = 0;';
-                const online = await req.app.locals.query(query);
-                return res.status(200).send(online[0].ct.toString());
-            }
-            res.status(404).send();
-        });
+        let gameServerOnline = false;
+        if (process.env.MOCK_GAME_SERVER_ONLINE == "true") {
```

Comment:
Can we use strict equality here?

Reply:
I know there was someone adding linting to the client, but for now no linting here. Can we also use single quotes here?